osmnx/core.py

```diff
@@ -183,14 +183,15 @@ def get_pause_duration(recursive_delay=5, default_duration=10):
     -------
     int
     """
+
     try:
-        response = requests.get('http://overpass-api.de/api/status', headers=get_http_headers())
+        response = requests.get(settings.overpass_endpoint.replace('interpteter', 'status'), headers=get_http_headers())
```

Comment:
It looks like there is a typo here: interpteter should be interpreter. Same on line 194